packages/desktop-client/src/components/transactions/TransactionsTable.tsx

```diff
@@ -446,9 +421,9 @@ function HeaderCell({
   marginRight,
   icon,
   onClick,
-}) {
+}: HeaderCellProps) {
   const style = {
-    whiteSpace: 'nowrap',
+    whiteSpace: 'nowrap' as CSSProperties['whiteSpace'],
```

Comment:
Instead of a cast, does it work to annotate the `style` variable as `Partial<CSSProperties>`?

Reply:
Yep, that works, I'll switch it to that